backend/lcfs/web/api/fuel_code/views.py

```diff
@@ -162,53 +141,9 @@ async def delete_fuel_code(
 ):
     return await service.delete_fuel_code(fuel_code_id)
 
-
-@router.get(
-    "/energy-densities",
-    response_model=List[EnergyDensitySchema],
-    status_code=status.HTTP_200_OK,
-)
-@view_handler(["*"])
-async def get_energy_densities(
-    request: Request,
-    service: FuelCodeServices = Depends(),
-):
-    """Endpoint to get energy densities"""
-    return await service.get_energy_densities()
-
-
-@router.get(
-    "/energy-effectiveness-ratios",
-    response_model=List[EnergyEffectivenessRatioSchema],
-    status_code=status.HTTP_200_OK,
-)
-@view_handler(["*"])
-async def get_energy_effectiveness_ratios(
-    request: Request,
-    service: FuelCodeServices = Depends(),
-):
-    """Endpoint to get energy effectiveness ratios (EER)"""
-    return await service.get_energy_effectiveness_ratios()
-
-
-@router.get(
-    "/additional-carbon-intensities",
-    response_model=List[AdditionalCarbonIntensitySchema],
-    status_code=status.HTTP_200_OK,
-)
-@view_handler(["*"])
-async def get_use_of_a_carbon_intensities(
-    request: Request,
-    service: FuelCodeServices = Depends(),
-):
-    """Endpoint to get UCI's"""
-    return await service.get_use_of_a_carbon_intensities()
-
-
 @router.post(
     "/save",
-    response_model=Union[FuelCodeSchema, DeleteFuelCodeResponseSchema],
-    status_code=status.HTTP_200_OK,
```

Comment:
Whats the reason for removing the status_code? What do these actually do?

Reply:
fixed.